dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageChecker.java

```diff
@@ -456,6 +456,14 @@ private boolean isMemberAccessibleFrom(
     String sourceClassName = classFile.getBinaryName();
     String targetClassName = symbol.getClassBinaryName();
 
+    // Skip references from Java runtime class. For example,
+    // com.sun.tools.internal.ws.wscompile.WsgenOptionsr referencing
+    // com.sun.xml.internal.ws.api.BindingID$SOAPHTTPImpl.
+    // https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1599
+    if (classDumper.isSystemClass(sourceClassName)) {
```

Comment:
sourceClassName `com.sun.tools.internal.ws.wscompile.WsgenOptionsr` returns false for `isSystemClass` as it's loaded by AppClassLoader. The class uses the same class loader as dependencies in pom.xml